style.css

```diff
@@ -531,3 +531,34 @@ body > * > * {
   margin-top: -7px;
   border-width: 7px 7px 7px 0;
 }
+
+details {
+  font: 16px;
+  width: 1000px;
```

Comment:
I wonder why we use this width rather than using the full page width. Doesn't seem to align with any of the other elements.